crates/holochain/src/core/workflow/sys_validation_workflow/test_ideas.rs

```diff
@@ -0,0 +1,14 @@
+/// Stubs for things that might break validation
+
+/// This test shows a way to create a delete with a rejected header
+/// and keep it in an agents cache.
+/// The same test can be done with an update.
+#[test]
+fn get_rejected_delete_into_cache_as_valid() {
```

Comment:
```suggestion
#[test]
#[ignore = "stub"]
fn get_rejected_delete_into_cache_as_valid() {
```

otherwise we won't know this test isn't implemented yet